docs/migrations/v6-to-v7.md

```diff
@@ -45,6 +45,8 @@ The `CheckMisbehaviourAndUpdateState` function has been removed from `ClientStat
 
 The function `GetTimestampAtHeight` has been added to the `ClientState` interface. It should return the timestamp for a consensus state associated with the provided height.
 
+A zero proof height is now allowed by core IBC and may be passed into `VerifyMembership` and `VerifyNonMembership`. Light clients are responsible for returning an error if a zero proof height is invalid behaviour. 
```

Comment:
Should we also mention relayers are must pass zero height for solomachines?

Reply:
Relayers can pass non zero heights. Solo machine simply ignores the provided value